### PR TITLE
Versioncheck

### PR DESCRIPTION
**Pull request by chuckries**
Implement a version check of the dotnet cli tools on the path. Give a good error message if the tools are too old.

### CODE REVIEW

File: src/coreclr-debug/activate.ts [hunk 1]
```diff
@@ -61,54 +54,98 @@ export function activate(context: vscode.ExtensionContext, reporter: TelemetryRe
...
+    }).catch((error) => {
+        // something went wrong with spawning 'dotnet --info'
+        let message = 'The .NET CLI tools cannot be located. .NET Core debugging will not be enabled. Make sure .NET CLI tools are installed and are on the path.';
+        showDotnetToolsWarning(message);
+        throw new Error("Failed to spawn 'dotnet --info'");
+    }).then(() => {
+        // succesfully spawned 'dotnet --info', check the Version
+        if (semver.lt(dotnetInfo.Version, MINIMUM_SUPPORTED_DOTNET_CLI))
+        {
+            let message = 'The .NET CLI tools on the path are too old. .NET Core debugging will ont be enabled. The minimum supported version is ' + MINIMUM_SUPPORTED_DOTNET_CLI + '. ';
```
Comment (chuckries) [inline src/coreclr-debug/activate.ts]: typo.

**Comment by chuckries** [comment]
@gregg-miskelly @wesrupert @caslan

**Comment by wesrupert** [comment]
LGTM! :shipit:

**Comment by gregg-miskelly** [comment]
LGTM

**Comment by chuckries** [comment]
Fixes #618

**Pull request merged**